k8s/charts/openebs/templates/deployment-maya-apiserver.yaml

```diff
@@ -21,9 +21,9 @@ spec:
         ports: 
         - containerPort: 5656
         env:
-        - name: DEFAULT_CONTROLLER_IMAGE
+        - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE
```

Comment:
This is all OK as these will not change in a release.

Reply:
this key is specific to 0.5.0 release. so we do need to make sure we dont break the backward compatibility, however to avoid such errors in future, i think we will need to auto generate the yaml files through a container -- which can take an yaml file with all the customizations.